app/views/admin/dashboard/show.html.erb

```diff
@@ -0,0 +1,59 @@
+<h2> Orders </h2>
+
+<p><h3> Packaged </h3></p>
+
+<section id = "packaged">
+<% @orders.each do |order| %>
+  <section id = "packaged-order<%= order.id %>">
+    <% if order.status == "packaged" %>
+      <p>Name: <%=link_to "#{order.name}", "/admin/profile/#{order.user.id}" %></p>
+      <p>Order ID: <%= order.id %></p>
+      <p>Date Created: <%= order.created_at %></p>
+      <p><%= button_to "Ship this Order", "/admin/orders/#{order.id}", method: :patch %></p>
+    <% end %>
+  </section>
+<% end %>
+</section>
```

Comment:
Each section follows the same logic. Displays the name, order id, and date created for whichever orders match that section's conditional (packaged, pending, shipped, or cancelled).
The name is a link to that user's  admin show page